diag_manager/fms_diag_axis_object.F90

```diff
@@ -0,0 +1,287 @@
+!***********************************************************************
+!*                   GNU Lesser General Public License
+!*
+!* This file is part of the GFDL Flexible Modeling System (FMS).
+!*
+!* FMS is free software: you can redistribute it and/or modify it under
+!* the terms of the GNU Lesser General Public License as published by
+!* the Free Software Foundation, either version 3 of the License, or (at
+!* your option) any later version.
+!*
+!* FMS is distributed in the hope that it will be useful, but WITHOUT
+!* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+!* FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+!* for more details.
+!*
+!* You should have received a copy of the GNU Lesser General Public
+!* License along with FMS.  If not, see <http://www.gnu.org/licenses/>.
+!***********************************************************************
+
+!> @defgroup fms_diag_axis_object_mod fms_diag_axis_object_mod
+!> @ingroup diag_manager
+!! @brief fms_diag_axis_object_mod stores the diag_axis_object
+
+!> @file
+!> @brief File for @ref diag_axis_object_mod
+
+!> @addtogroup fms_diag_axis_object_mod
+!> @{
+module fms_diag_axis_object_mod
+  use mpp_domains_mod, only:  domain1d, domain2d, domainUG, mpp_get_compute_domain, CENTER, &
```

Comment:
What about `NORTH` and `EAST`?

Reply:
They are not being used... It might be used later, but not right now. 
Center is just used to initialize the %domain_position variable:
https://github.com/NOAA-GFDL/FMS/blob/f8610606f52991c97010d5bb79395af6e3dbad25/diag_manager/fms_diag_axis_object.F90#L173

Reply:
They will definitely be used, but you can add them when needed.